docs/reference/alpha-on-demand-feature-view.md

```diff
@@ -25,19 +25,20 @@ See [https://github.com/feast-dev/on-demand-feature-views-demo](https://github.c
 
 ### **Registering transformations**
 
-We register `RequestDataSource` inputs and the transform in `on_demand_feature_view`:
+We register `RequestSource` inputs and the transform in `on_demand_feature_view`:
 
 ```python
 from feast import Field, RequestSource
-from feast.types import Float64
+from feast.types import Float64, Int64
+import pandas as pd
 
 # Define a request data source which encodes features / information only 
 # available at request time (e.g. part of the user initiated HTTP request)
-input_request = RequestDataSource(
+input_request = RequestSource(
     name="vals_to_add",
     schema={
-        "val_to_add": ValueType.INT64,
-        "val_to_add_2": ValueType.INT64
+        "val_to_add": Int64,
+        "val_to_add_2": Int64
     }
```

Comment:
nit: this should be [Field(name="val_to_add", dtype=Int64)...]

Reply:
nice catch. Thanks!